src/Utils.H

```diff
@@ -11,42 +15,50 @@
 namespace ExaEpi
 {
 
+/*! \brief Structure with various simulation parameters; these are read in
+    for the input file provided in the command line.
+*/
 struct TestParams
 {
-    amrex::IntVect size;
-    int max_grid_size;
-    int nsteps;
-    int plot_int;
-    int random_travel_int;
-    short ic_type;
-    std::string census_filename;
-    std::string workerflow_filename;
-    std::string case_filename;
-
-    int aggregated_diag_int;
-    std::string aggregated_diag_prefix;
+    amrex::IntVect size;                /*!< domain size; (QDG: used only for ic_type = demo?)
```

Comment:
@atmyers @tannguyen153 Is my comment correct (that the domain size specified in the inputs is used only when IC type is demo)?